cpu/mips_pic32mz/Makefile.include

```diff
@@ -0,0 +1,35 @@
+export MEMORY_BASE=0x80000000
+export MEMORY_SIZE=512K
+export APP_START=0x80000000
+export ROMABLE = 1
+
```

Comment:
For instance, this exports can be in a ld file located on the ldscripts folder for this cpu. See above.